### ISSUE TITLE
window size not opening correctly

### ISSUE THREAD

**Issue by rnhaddad** [eteran/nedit-ng#158]
I could not find this issue (i think there was one related to this already), so i am creating a new one:

First some info:
git show
commit e6c55dceb6ab718006d86a88db2f0ce325c31a50 (HEAD -> master, origin/master, origin/HEAD)

config.ini (relevant params):
nedit.textCols=80
nedit.textRows=50

cat /etc/os-release 
NAME="Ubuntu"
VERSION="18.04.4 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.4 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

Problem Description:
nedit Makefile &

The window size here is small (14x5) instead of 80x50 as configured.

Note that, 1 out of 10 times, the window size is open as desired i.e. 80x50.
If i resize the small window (drag the corners), then close the window, then with a high probability the window size re-opens with 80x50 but not always.

The problem seem to be "random", and/or due to some race condition.

**Comment by eteran** [comment]
Are you using `nc` by any chance? I think I recall battling a similar bug in the past.

**Comment by anjohnson** [comment]
@eteran I don't think you ever actually fixed the original bug with `nc-ng` which I suspect is some kind of race condition that also may be responsible for the occasional result that files open in new windows instead of as new tabs in an existing server window. That behavior I see on the Mac as well as on RHEL (if I open 2 files with it I often get 2 new windows). I haven't seen this window-size problem on the Mac though.

**Comment by eteran** [comment]
Yea, that issue IIRC is **really** tricky, I'll have to revisit it sometime to see if a fresh set of eyes can figure it out.
